pkg/certificates/ca_test.go

```diff
@@ -162,7 +162,7 @@ rydzkVNNVeMVX2TER9yc8AdFqkRlaBWHmO61rYmV+N1quLM0uMVsu55ZNCY=
 			mockRsa := mock_certificates.NewMockRsaer(ctrl)
 
 			mockRsa.EXPECT().GenerateKey(gomock.Any(), gomock.Any()).DoAndReturn(
-				func(random io.Reader, bits int) (*rsa.PrivateKey, error) {
+				func(random io.Reader, bits int) (*rsa.PrivateKey, error) { //nolint:revive
```

Comment:
can the `//nolint:revive` be added globally?

Reply:
As we discussed, this involved disabling revive in the configuration